### PR TITLE
Add N&N for .target's new "Follow repository references" option

### PR DESCRIPTION
**Pull request by tivervac**
https://github.com/eclipse-pde/eclipse.pde/pull/1253/files

### CODE REVIEW

**Comment by tivervac** [comment]
I'm not sure who I ping best, @merks could you approve, please?

File: news/4.32/pde.html [hunk 1]
```diff
@@ -76,11 +76,18 @@ <h2>Plug-in Development Environment</h2>
       </td>
   </tr>
   
-  <!-- ******************** End ofDialogs, Wizard and Views ********************** -->
+  <!-- ******************** End of Dialogs, Wizard and Views ********************** -->
   
   <!-- ******************** Editors ********************** -->
   <tr>
     <td id="editors" class="section" colspan="2"><h2>Editors</h2></td>
+    <td class="content">
```
Comment (iloveeclipse) [inline news/4.32/pde.html:84]: Please add comment with the link to the original issue fixed, like in the section above

Comment (tivervac) [reply news/4.32/pde.html:84]: @iloveeclipse Sorry, I lost sight of this. Added it now

File: news/4.32/pde.html [hunk 2]
```diff
@@ -76,11 +76,19 @@ <h2>Plug-in Development Environment</h2>
       </td>
   </tr>
   
-  <!-- ******************** End ofDialogs, Wizard and Views ********************** -->
+  <!-- ******************** End of Dialogs, Wizard and Views ********************** -->
   
   <!-- ******************** Editors ********************** -->
   <tr>
     <td id="editors" class="section" colspan="2"><h2>Editors</h2></td>
+    <!-- https://github.com/eclipse-pde/eclipse.pde/pull/1253/files -->
```
Comment (tivervac) [inline news/4.32/pde.html:84]:
```suggestion
    <!-- https://github.com/eclipse-pde/eclipse.pde/pull/1253 -->
```

**Comment by merks** [comment]
This is website change.  I'll merge it now.

**Pull request merged**